### ISSUE TITLE
这里计算pred_y是不是有问题？

### ISSUE THREAD

**Issue by gwd777** [teo-sl/DPLAN_pytorch#4]
pred_y只是取【a0，a1】当中的a1，不应该是取max(a0,a1)嘛？如果这样处理，那么新来的一个样本，如何确定它是否有缺陷呢？

def test_model(test_set,policy_net):
    policy_net.eval()
    test_X, test_y=test_set[:,:-1], test_set[:,-1]
    pred_y=policy_net(test_X).detach().cpu().numpy()[:,1]

    roc = roc_auc_score(test_y, pred_y)
    pr = average_precision_score(test_y, pred_y)
    policy_net.train()
    return roc,pr
